src/System.Private.CoreLib/shared/System/Math.cs

```diff
@@ -0,0 +1,836 @@
+        }
+
+        public static int Sign(int value)
+        {
+            return unchecked(value >> 31 | (int)((uint)-value >> 31));
+        }
+
+        public static int Sign(long value)
+        {
+            return unchecked((int)(value >> 63 | (long)((ulong)-value >> 63)));
+        }
+
+        [CLSCompliant(false)]
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static int Sign(sbyte value)
+        {
+            return Sign((int)value);
+        }
+
+        public static int Sign(float value)
+        {
+            if (value < 0)
+            {
+                return -1;
+            }
+            else if (value > 0)
+            {
+                return 1;
+            }
+            else if (value == 0)
+            {
+                return 0;
+            }
+
+            throw new ArithmeticException(SR.Arithmetic_NaN);
+        }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static decimal Truncate(decimal d)
+        {
+            return decimal.Truncate(d);
+        }
+
+        public static unsafe double Truncate(double d)
+        {
+            modf(d, &d);
+            return d;
+        }
+
+        private static short AbsHelper(short value)
+        {
+            Contract.Requires(value < 0, "AbsHelper should only be called for negative values! (workaround for JIT inlining)");
+
+            if (value == short.MinValue)
+            {
+                throw new OverflowException(SR.Overflow_NegateTwosCompNum);
+            }
+            Contract.EndContractBlock();
+
+            return ((short)(-value));
+        }
+
+        private static int AbsHelper(int value)
+        {
+            Contract.Requires(value < 0, "AbsHelper should only be called for negative values! (workaround for JIT inlining)");
+
+            if (value == int.MinValue)
+            {
+                throw new OverflowException(SR.Overflow_NegateTwosCompNum);
+            }
+            Contract.EndContractBlock();
+
+            return -value;
+        }
+
+        private static long AbsHelper(long value)
+        {
+            Contract.Requires(value < 0, "AbsHelper should only be called for negative values! (workaround for JIT inlining)");
+
+            if (value == long.MinValue)
+            {
+                throw new OverflowException(SR.Overflow_NegateTwosCompNum);
+            }
+            Contract.EndContractBlock();
+
+            return -value;
+        }
+
+        private static sbyte AbsHelper(sbyte value)
+        {
+            Contract.Requires(value < 0, "AbsHelper should only be called for negative values! (workaround for JIT inlining)");
+
+            if (value == sbyte.MinValue)
+            {
+                throw new OverflowException(SR.Overflow_NegateTwosCompNum);
+            }
+            Contract.EndContractBlock();
+
+            return ((sbyte)(-value));
+        }
+
+        private static unsafe double copysign(double x, double y)
```

Comment:
Looks like I missed fixing the casing of this in the CoreCLR PR.

Reply:
You can append commit to fix this to this PR if you would like.